pkgs/by-name/to/tone/package.nix

```diff
@@ -1,33 +1,46 @@
-{ lib, fetchFromGitHub, buildDotnetModule, ffmpeg-full, dotnetCorePackages }:
+{
+  lib,
+  fetchFromGitHub,
+  buildDotnetModule,
+  ffmpeg-full,
+  stdenv,
+  versionCheckHook,
+}:
 
 buildDotnetModule rec {
   pname = "tone";
-  version = "0.1.5";
+  version = "0.2.3";
 
   src = fetchFromGitHub {
     owner = "sandreas";
-    repo = pname;
+    repo = "tone";
     rev = "v${version}";
-    sha256 = "sha256-HhXyOPoDtraT7ef0kpE7SCQbvGFLrTddzS6Kdu0LxW4=";
+    hash = "sha256-fe0fcX617vsRdu42FEan181Zh5Todt9FLtOuD+C7yqA=";
```

Comment:
I got a hash mismatch

```
error: hash mismatch in fixed-output derivation '/nix/store/9y5bgmn5il6fpwkgzhwi8sld7i6fpvi9-source.drv':
        likely URL: https://github.com/sandreas/tone/archive/v0.2.3.tar.gz
         specified: sha256-fe0fcX617vsRdu42FEan181Zh5Todt9FLtOuD+C7yqA=
            got:    sha256-NBFAPEeUKZgyfNlvcOBS1IpktEnI+fOd9WLj0ByzpLY=
```

Reply:
Yeah, I think that's what happened. Should be good now.